### DISCUSSION TITLE
Intelowl & MISP integration health failed

### DISCUSSION THREAD

**Discussion by v3daxt** [intelowlproject/IntelOwl#850] [Q&A]
I need to feed my IntelOwl data to MISP. I am running both of them on two different kali machines since both of them are using docker and can not be used on same machine.

I have done following configurations on `connector_config.js` and `env_fle_app` file for MISP connector
```
"MISP" "disabled": false
MISP_KEY="My_misp_instance_key"
CONNECTOR_MISP_KEY="My_misp_instance_key"
MISP_URL=https://localhost
CONNECTOR_MISP_URL=https://localhost
```

I am not sure what MISP_URL should be set since my MISP instance is running on different vm machine
In intelowl I  am getting configured = green tick but for health check, my status is failed

**Comment by eshaan7** [comment]
I think you just need a way to port-forward the port your MISP instance is running on from the MISP VM box to IntelOwl VM box. Depending on whether you are using virtualbox or vm-ware, you should be able to find information on this.
